### PR TITLE
TDA Products Join

### PR DESCRIPTION
**Pull request by thinkocapo**
Test that you can run TDA on the /products-join route and make any necessary updates.

#### Overview
This worked, the test is only getting the /products-join endpoint, it does not add items to the shopping cart, but that's okay.
```
driver.get("http://application-monitoring-react-dot-sales-engineering-sf.appspot.com/products-join")
```

### CODE REVIEW

**Review by ndmanvar — APPROVED** [review]

**Comment by ndmanvar** [comment]
looks good + straightforward!

**Comment by thinkocapo** [comment]
@ndmanvar Tested - I ran it from the compute engine, it produced transactions for /products and /products-join in near equal amounts, which was expected.
image

File: app.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
   
 env: standard
-service: application-monitoring-react
+service: -<SERVICE_NAME_HERE>-
```
Comment (thinkocapo) [inline app.yaml:3]: this change helps prevent people from accidentally overwriting the master admin instance.

**Comment by thinkocapo** [comment]
Noting the difference in `products` vs `products_join` after running TDA for nearly 24 hours
image

**Pull request merged**